libs/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -856,6 +853,82 @@ def add_texts(
             logger.debug("Nothing to insert, skipping.")
             return []
 
+        transposed_embeddings = [
+            [embeddings[j][i] for j in range(len(embeddings))]
+            for i in range(len(embeddings[0]))
+        ]
+        # Now:
+        # transposed_embeddings = [
+        #     [f1(a), f2(a)],
+        #     [f1(b), f2(b)],
+        #     [f1(c), f2(c)]
+        # ]
+
+        return self.add_embeddings(
+            texts=texts,
+            embeddings=transposed_embeddings,
+            metadatas=metadatas,
+            timeout=timeout,
+            batch_size=batch_size,
+            ids=ids,
+            **kwargs,
+        )
+
+    def add_embeddings(
+        self,
+        texts: List[str],
+        embeddings: Union[List[List[float]], List[List[List[float]]]],
```

Comment:
It's was not trivial to do `zip(text, embeddings)` as in the multi-vector settings actually the "layout" is different as described above